src/ucp/api/ucp_def.h

```diff
@@ -302,6 +324,32 @@ typedef struct ucp_err_handler {
 
 
 /**
+ * @ingroup UCP_EP
+ * @brief UCP address description in case of a client-server connection flow.
```

Comment:
right, need to:
1. remote "@ingroup UCP_EP" since this is used for ucp_worker_listen() as well
2. i would phrase it something like "BSD socket address specification". The doc should describe what it _is_, not how it's currently used.